interp/constrintern.mli

```diff
@@ -227,3 +227,7 @@ val interp_cumul_univ_decl_opt : Environ.env -> cumul_univ_decl_expr option ->
   Evd.evar_map * UState.universe_decl * Entries.variance_entry
 (** BEWARE the variance entry needs to be adjusted by
    [ComInductive.variance_of_entry] if the instance is extensible. *)
+
+val interp_mutual_univ_decl_opt : Environ.env -> universe_decl_expr option list ->
+                       Evd.evar_map * UState.universe_decl
```

Comment:
too indented

Reply:
ping

Reply:
@herbelin can you see this comment?

Reply:
Now yes, thanks for the reminder. Fixed (and ml comment rephrased).